FIAT/functional.py

```diff
@@ -123,7 +129,25 @@ def to_riesz(self, poly_set):
                     result[c][i] += w * bfs[i, j]
 
         if self.deriv_dict:
-            raise NotImplementedError("Generic to_riesz implementation does not support derivatives")
+            dpt_dict = self.deriv_dict
+
+            # this makes things quicker since it uses dmats after
+            # instantiation
+            es_foo = polynomial_set.ONPolynomialSet(self.ref_el, ed) 
+            dpts = list(dpt_dict.keys())
+
+            # figure out the maximum order of derivative being taken
+            dbfs = es_foo.tabulate(dpts, self.max_deriv_order)
+
+            ndpts = len(dpts)
+            for j in range(ndpts):
+                dpt_cur = dpts[j]
+                wac_list = dpt_dict[dpt_cur]
+                for i in range(nbf):
+                    for (w, alpha, c) in wac_list:
+                        result[c][i] += w * dbfs[alpha][i, j]
+
+            #raise NotImplementedError("Generic to_riesz implementation does not support derivatives")
```

Comment:
left over debugging or intended use elsewhere?

Reply:
Thanks for taking a look.  I changed this as we..